action-types.yml

```diff
@@ -0,0 +1,11 @@
+# See https://github.com/krzema12/github-actions-typing
+
+inputs:
```

Comment:
Can I ask how this is different from `action.yml` file?

Reply:
It's just a designated file for the typings. Initially they were in action.yml, but I received a feedback that formally the typings don't fit action.yml's schema. That's why this separate file was created.

Reply:
@bahmutov does it make sense to you?

Reply:
I have decided to ultimately decline this PR, since I think the typings should be part of the GitHub Actions validations rather than userspace, and if Actions don't have it yet, I don't want to maintain them separately. Thank you for opening the PR, though